docs/src/pages/components/Switch.vue

```diff
@@ -36,6 +36,16 @@
           <md-switch v-model="checked7" id="my-test7" name="my-test7" class="md-primary" disabled>Light Blue Primary Color Disabled</md-switch>
         </div>
       </demo-example>
+
+      <demo-example label="Typed">
```

Comment:
Maybe this typed section can have the ```submit``` type instead of ```button```, as this is the default behaviour.